src/ompl/util/src/RandomNumbers.cpp

```diff
@@ -44,6 +47,24 @@
 #include <boost/scoped_ptr.hpp>
 #include <boost/date_time/posix_time/posix_time.hpp>
 #include <boost/math/constants/constants.hpp>
+#include <boost/random/uniform_on_sphere.hpp>
+// For boost::make_shared
+#include <boost/make_shared.hpp>
+// For pre C++ 11 gamma function
+#include <boost/math/special_functions/gamma.hpp>
+// For boost::numeric::ublas::shallow_array_adaptor:
+#include <boost/numeric/ublas/vector.hpp>
+
+// The Eigen Includes:
+// The core
+#include <Eigen/Core>
```

Comment:
I don't have any real insight into this. I know Eigen is obviously popular in ROS, and I think it's also pretty common in the estimation community.

I had actually asked on the mailing list before starting, but that was almost a year ago... :blush:
http://sourceforge.net/p/ompl/mailman/ompl-users/thread/5386075D.2090608%40robotics.utias.utoronto.ca/#msg32391199